projects/angular-ui/src/lib/button/button.component.ts

```diff
@@ -97,13 +97,18 @@ export class BaoButtonComponent implements AfterViewInit {
   public ngAfterViewInit() {
```

Comment:
Thanks for submitting this PR. Did you try to replace ngAfterViewInit by ngAfterContentInit

Reply:
This is indeed a better solution. I tested it and it works well. I updated the PR